cdap-docs/developers-manual/source/building-blocks/metadata-lineage.rst

```diff
@@ -2,14 +2,35 @@
     :author: Cask Data, Inc.
     :copyright: Copyright © 2015 Cask Data, Inc.
 
-.. _metadata:
+.. _metadata-lineage:
 
-========
-Metadata
-========
+====================
+Metadata and Lineage
+====================
 
 Overview
 ========
+Metadata and Lineage are a new and important feature of CDAP. CDAP Metadata helps show how
+datasets and programs are related to each other and helps in understanding the impact of a
+change before the change is made. 
+
+This feature provides full visibility into the impact of changes while providing an audit
+trail of access to datasets by programs and applications. It gives a clear view when
+identifying trusted data sources and enables the ability to track the trail of sensitive
+data.
+
+CDAP captures metadata from many different sources |---| as well as those specified by a
+user |---| on different entities and objects. The container model of CDAP provides for the
+seamless aggregation of a wide variety of machine-generated metadata that is automatically
```

Comment:
We associate Workflow tokens and such with dataset through the runid that are generated - So isn't it associating machine generated metadata ?